SPECS/xapi.spec

```diff
@@ -165,6 +165,11 @@ Conflicts: secureboot-certificates < 1.0.0-1
 Conflicts: varstored < 1.2.0-1
 BuildRequires: systemd
 %{?systemd_requires}
+%if 0%{?xenserver} < 9
+# sysprep plugin/API
+Requires: genisoimage
```

Comment:
Do we want to enable the sysprep plugin?

Reply:
any reason not to?

Reply:
I don't want it to cause unneeded failures, but maybe it's OK because this needs to enabled from XO for each VM

Reply:
it's "a new API call that works in collaboration with a guest agent on a Windows VM to pass a unattend.xml to Windows sysprep." - https://github.com/xapi-project/xen-api/pull/6547

Reply:
Ok, I remember this from a previous discussion. Did we conclude that it could be useful to our users, @dinhngtu?